app/routes.py

```diff
@@ -218,7 +218,16 @@ def location_search_results():
     )
 
 
-@main_bp.route("/create-masterclass/location/details", methods=["GET"])
+@main_bp.route("/create-masterclass/location/details", methods=["GET", "POST"])
 @login_required
 def add_in_person_location_details():
+    if request.method == "POST":
+        draft_masterclass = Masterclass.query.get(session["draft_masterclass_id"])
```

Comment:
see previous comment about thinning the controller - we'll discuss approaches at our next session